src/tasks/minions/minigames/raidsActivity.ts

```diff
@@ -108,5 +110,30 @@ export default class extends Task {
 		);
 
 		sendToChannelID(this.client, channelID, { content: resultMessage });
+
+		if (allUsers.length === 1) {
+			handleTripFinish(
+				this.client,
+				allUsers[0],
+				channelID,
+				'',
+				res => {
+					const flags: Record<string, string> = challengeMode ? { cm: 'cm' } : {};
+
+					// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+					// @ts-ignore
+					if (!res.prompter) res.prompter = {};
+					// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+					// @ts-ignore
+					res.prompter.flags = flags;
+
+					allUsers[0].log('continued trip of solo CoX');
+					return this.client.commands.get('raid')!.run(res, ['solo']);
+				},
+				undefined,
+				data,
+				null
+			);
+		}
 	}
```

Comment:
Isn't this going to cause 2 messages to be sent? you should do only sendtochannelid in masses, only handetripfinish in solos.

Also, please don't make PRs like this with lots of unrelated changes. Making 10 different PRs with small changes is a lot easier.

Reply:
Right you are on both counts! Future PRs won't have the same issue.